### PR TITLE
expression: implement vectorized evaluation for 'builtinNameConstDurationSig'

### PR DESCRIPTION
**Pull request by tsthght**
### What problem does this PR solve? 
Implement vectorized evaluation for builtinNameConstDurationSig, for #12176

### What is changed and how it works?
```
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinNameConstDurationSig-VecBuiltinFunc-8         	20000000	       115 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinNameConstDurationSig-NonVecBuiltinFunc-8      	  100000	     21252 ns/op	       0 B/op	       0 allocs/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: expression/builtin_miscellaneous_vec.go [hunk 1]
```diff
@@ -145,11 +145,14 @@ func (b *builtinUUIDSig) vecEvalString(input *chunk.Chunk, result *chunk.Column)
...
-	return false
+	return true
 }
 
 func (b *builtinNameConstDurationSig) vecEvalDuration(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[1].VecEvalDuration(b.ctx, input, result); err != nil {
+		return err
+	}
+	return nil
```
Comment (SunRunAway) [inline expression/builtin_miscellaneous_vec.go]: Use `return b.args[1].VecEvalDuration(b.ctx, input, result)` for shortly?

Comment (tsthght) [reply expression/builtin_miscellaneous_vec.go]: ok~

**Pull request merged**